src/main/resources/application.yml

```diff
@@ -23,3 +22,12 @@ spring:
 self-order:
   admin:
     access-token: Z0VdRIeUZzHTFAy9n2kRc=w=tUZikgaUyO6s64sO986g-5uWestRONqUJbHxTxw821/We-DMe9lxEuKkuzCTW=WJ!R8SrC9cI8oXfcc!BbZOQ21QFvj92?sjEIGRjyWbWhrb0rMl5/2TBHE?x6FS2hd9JZdcR9UQzTak?wan-4LMPnOvTrhlSHRFDhXzyFOojo/SnjbwkMEaO3GJ6FIU0?jLC-rLuvtzUNcT9KU0TrLMFmS2m03PvKRzlO5=qeqc
+
+mercadopago:
+  api:
+    url: https://api.mercadopago.com
+    token: REPLACE
+    userId: REPLACE
+  integration:
+    posId: REPLACE
+    webhook: REPLACE
```

Comment:
Faça uma conta no Mercado Pago e crie uma conta de teste como vendedor. Faça login nessa conta de teste de vendedor e colete o access token e o user ID.

Para `posId`, o ID do caixa (Point of Sale), siga a documentação adicionada nesse PR.

Como webhook, nessa parte 1, use ferramentas como `https://webhook.site`.